platform/linux-generic/odp_ishm.c

```diff
@@ -1645,7 +1645,7 @@ int _odp_ishm_cleanup_files(const char *dirpath)
 				return -1;
 			}
 			snprintf(fullpath, PATH_MAX, "%s/%s",
-				 dirpath, e->d_name);
+				 userdir, e->d_name);
```

Comment:
A good catch. The following line above: `fullpath = malloc(d_len + f_len + 2);` also needs to be fixed. Currently, `d_len` refers to `dirpath`.

Reply:
A good review.

Reply:
Hi @chrhong, any progress with signing the  CLA?